controller/rest/system.go

```diff
@@ -2116,6 +2117,223 @@ func handlerSystemGetRBAC(w http.ResponseWriter, r *http.Request, ps httprouter.
 	restRespSuccess(w, r, &resp, acc, login, nil, "Get missing Kubernetes RBAC")
 }
 
+func getAlertGroup(alerts []string, alertType api.AlertType, acceptedAlerts utils.Set) (*api.RESTNvAlertGroup, error) {
+	alertGroup := &api.RESTNvAlertGroup{
+		Type: alertType,
+	}
+
+	if len(alerts) > 0 {
+		for _, alert := range alerts {
+			b := md5.Sum([]byte(alert))
+			key := hex.EncodeToString(b[:])
+			if !acceptedAlerts.Contains(key) {
+				alertGroup.Data = append(alertGroup.Data, &api.RESTNvAlert{
+					ID:      key,
+					Message: alert,
+				})
+			}
+		}
+	}
+
+	if len(alertGroup.Data) > 0 {
+		return alertGroup, nil
+	}
+
+	return alertGroup, errors.New("no alert is retrieved")
```

Comment:
Do we consider the case with no alerts as an error?  How about this?
```suggestion
	return nil, nil
```

Reply:
I was thinking to use error to filter the case to pass the alertGroup with data. But it makes more sense to me to return nil, and in this case, we don't even need error :P